guides/common/modules/proc_disabling-subscription-connection.adoc

```diff
@@ -0,0 +1,19 @@
+[id="disabling-subscription-connection_{context}"]
+= Disabling Subscription Connection
+
+Disable subscription connection on {ProjectServer} to avoid attempts to consume content from the Red Hat Content Delivery Network (CDN).
```

Comment:
```suggestion
On a disconnected server disable subscription connection on {ProjectServer} to avoid connecting to the Red Hat Portal. This will prevent you from refreshing the manifest, updating upstream entitlements and ability to change SCA status.
```